connectors/src/connectors/slack/chat/blocks.ts

```diff
@@ -47,14 +46,18 @@ function makeDividerBlock() {
   };
 }
 
-function makeMarkdownBlock(text: string) {
-  return {
-    type: "section",
-    text: {
-      type: "mrkdwn",
-      text: truncate(text, MAX_SLACK_MESSAGE_LENGTH),
-    },
-  };
+function makeMarkdownBlock(text?: string) {
```

Comment:
If there is no text, shouldn't we keep displaying "_Thinking_"?

Reply:
we only show "Thinking" if isThinking is set, whatever there's text or not

Reply:
Shouldn't those be mutually exclusive?

Reply:
No, showing "thinking" should not hide the text previously generated